### DISCUSSION TITLE
model conversion to python

### DISCUSSION THREAD

**Discussion by mattiasospetti** [vladmandic/human#429] [General]
I am developing it in Python, but your main application is in JS.
Even though a porting from JS to Python could fit, I find it unfeasible, so I thought that I could start from the models.
May I ask you a question? Is it possible to convert the models (just the gender age emotion) from tfjs (bin and json) to tensorflow (for Python)?

Thanks to whomever will reply!

**Comment by vladmandic** [comment]
you can try conversion from tfjs graph model format to tf saved model format using https://github.com/patlevin/tfjs-to-tf/
